app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/view/store-pickup.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'uiComponent',
+    'ko',
+    'underscore',
+    'Magento_Checkout/js/model/quote',
+    'Magento_Checkout/js/action/select-shipping-method',
+    'Magento_Checkout/js/checkout-data',
+    'Magento_Checkout/js/model/shipping-service',
+    'Magento_Checkout/js/model/step-navigator',
+], function(
+    Component,
+    ko,
+    _,
+    quote,
+    selectShippingMethodAction,
+    checkoutData,
+    shippingService,
+    stepNavigator
+) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            template: 'Magento_InventoryInStorePickupFrontend/store-pickup',
+            deliveryMethodSelectorTemplate:
+                'Magento_InventoryInStorePickupFrontend/delivery-method-selector',
+        },
+        isVisible: ko.observable(false),
+        isSelected: ko.observable(false),
+        rates: shippingService.getShippingRates(),
+        inStoreMethod: null,
+
+        initialize: function() {
+            var self = this;
```

Comment:
This seems to be redundant at this point. Also, I'd suggest to bind `this` to the context whenever possible.

Reply:
Fixed.